### ISSUE TITLE
website: How to install XPack

### ISSUE THREAD

**Issue by thernstig** [xpack-dev-tools/xpack-dev-tools.github.io#2]
I noticed that https://xpack-dev-tools.github.io/docs/getting-started/ talks about xPack and what it does and how to install packages with it, but not how to install xPack itself.

I did find https://xpack.github.io/install/ though.

Should these two not be "merged" in a sense so that the more modern Docosaurus-based site contains the installation instructions of xPack itself?

**Comment by ilg-ul** [comment]
Yes, you are right, the top xpack-dev-tools site does not link to that page, I'll fix it.

I'm currently migrating all web sites to Docusaurus, and hopefully these issues will be resolved.

I'll let you know when ready, perhaps you can provide some further feedback.
